geode-old-versions/build.gradle

```diff
@@ -15,13 +15,16 @@
  * limitations under the License.
  */
 
-plugins {
-  id "de.undercouch.download" version "3.2.0"
-}
 
-import de.undercouch.gradle.tasks.download.Download
 disableMavenPublishing()
 
+repositories {
+  maven { url "https://plugins.gradle.org/m2/" }
+  maven {
+    url 'https://repository.apache.org/content/repositories/snapshots'
```

Comment:
Why did you add the snapshots repository? It seems like we should be testings against an official old version, not a nightly snapshot.